src/functions/runWorkspaceTasks.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import Project from '../Project';
+import type {Config} from '../types';
+
+type WorkspaceInfo = {
```

Comment:
Fine to leave here for now, but reckon we need a types.js file in /functions as well? Likely types like this will be shared I think.

Reply:
Add it when we need it